docs/features/LOGGING.md

```diff
@@ -56,3 +58,50 @@ output.elasticsearch:
 If you are running Crawler inside of a Docker container and you have set it up to write logs to a mounted volume, you can simply point the `paths` fields to the directory your logs are being written to.
 
 Check out [Filebeat's quickstart guide](https://www.elastic.co/guide/en/beats/filebeat/current/index.html) to learn more about how to get it installed and running.
+
+## Event Logs
+### What are event logs?
+Event logs are detailed logs that contain information regarding pertinent 'events' Crawler undertakes during normal operation. These events include, but are not limited to:
+
+- When a Crawl starts
+- When a URL is discovered
+- When a URL is extracted
+- When a Crawl ends
+
+On a practical level, Event logs are log messages. What differentiates them from typical log messages, however, is that they are not related to _system_ level logs. In other words, Event logs are not tied to the typical DEBUG, INFO, WARN, ERROR and FATAL levels, and oftentimes are not generated from the same places those logs get generated from.
```

Comment:
I think we need consistency with capitalisation. Either `Event Logs` or `event logs` (probably the latter?).